frontend/src/components/pages/connect/CreateConnector.tsx

```diff
@@ -144,218 +170,287 @@ function ConnectorWizard({ connectClusters }: ConnectorWizardProps) {
             title: 'Connector Type',
             description: 'Choose type of connector.',
             icon: <DatabaseOutlined />,
-            content: <ConnectorType
-                connectClusters={connectClusters}
-                activeCluster={activeCluster}
-                onActiveClusterChange={setActiveCluster}
-                selectedPlugin={selectedPlugin}
-                onPluginSelectionChange={setSelectedPlugin}
-            />,
+            content: (
+                <ConnectorType
+                    connectClusters={connectClusters}
+                    activeCluster={activeCluster}
+                    onActiveClusterChange={(clusterName) => {
+                        uiState.pageBreadcrumbs[1] = { title: clusterName!, linkTo: `/connect-clusters/${clusterName}` };
```

Comment:
Can we instead clear the breadcrumbs and push new entries instead modifying existing ones?
If its too hard, leave it as it is.
But I feel a bit uneasy about modifying array entries when there is no guarantee (array could be empty, or maybe there are additional entries or so...)